pkg/apis/kops/v1alpha1/networking.go

```diff
@@ -25,6 +25,7 @@ type NetworkingSpec struct {
 	Kopeio   *KopeioNetworkingSpec   `json:"kopeio,omitempty"`
 	Weave    *WeaveNetworkingSpec    `json:"weave,omitempty"`
 	Calico   *CalicoNetworkingSpec   `json:"calico,omitempty"`
+	Canal    *CanalNetworkingSpec    `json:"canal,omitempty"`
```

Comment:
Yep I followed that to get `conversion-gen` installed, but it seems to be puking on me when it runs